lib/gems/pending/appliance_console/cli.rb

```diff
@@ -182,13 +182,18 @@ def run
 
     def set_db
       raise "No encryption key (v2_key) present" unless key_configuration.key_exist?
+      raise "A password is required to configure a database" unless password?
       if local?
         set_internal_db
       else
         set_external_db
       end
     end
 
+    def password?
+      options[:password] && !options[:password].strip.empty?
+    end
+
```

Comment:
would below look more complicated or easer to read ? : 
`options[:password] ? !options[:password].strip.empty? : false`

Reply:
I think it's fine the way it is.

Reply:
Thanks @yrudman and @carbonin . Now should be good.